client.go

```diff
@@ -138,6 +138,20 @@ func New(address string, opts ...ClientOpt) (*Client, error) {
 	if copts.services == nil && c.conn == nil {
 		return nil, errors.New("no grpc connection or services is available")
 	}
+
+	// check namespace labels for default runtime
+	if copts.defaultRuntime == "" && copts.defaultns != "" {
```

Comment:
if `copts.defaultns == ""`, can we use `namespaces.Labels(ctx, namespaces.Default)`?

Reply:
Hey @AkihiroSuda, does that mean to use `default` namespace if `defaultns` is not passed? Because earlier I assumed that should be the case and @dmcgowan suggested not to do that in his review above.

Reply:
Please document that?

Reply:
Would this be the right place to that? https://github.com/containerd/containerd/blob/master/docs/namespaces.md